<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SeasonVarPirateQt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/logo/gfx/icon.png</normaloff>:/icons/logo/gfx/icon.png</iconset>
  </property>
  <widget class="QLabel" name="loadingAnimation">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>226</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/gfx/loading-animation.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeasonvarURL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Seasonvar URL: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSeasonvarURL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>441</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>441</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="openLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetAllTheLinks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Get all the links</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerateScript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Generate script</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDownloadFromLinks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Download from links</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCopyLinksToClipboard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Copy links to clipboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelParrot">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>369</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/icons/logo/gfx/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCloud">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/icons/logo/gfx/yarr.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLinksDatabase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Links database</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>182</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfigureProxy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Configure proxy</string>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>labelSeasonvarURL</zorder>
  <zorder>lineEditSeasonvarURL</zorder>
  <zorder>pushButtonGetAllTheLinks</zorder>
  <zorder>pushButtonGenerateScript</zorder>
  <zorder>pushButtonDownloadFromLinks</zorder>
  <zorder>pushButtonCopyLinksToClipboard</zorder>
  <zorder>labelParrot</zorder>
  <zorder>labelCloud</zorder>
  <zorder>loadingAnimation</zorder>
  <zorder>pushButtonLinksDatabase</zorder>
  <zorder>line</zorder>
  <zorder>pushButtonConfigureProxy</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
